packages/renderer/src/lib/engine/Prune.svelte

```diff
@@ -32,16 +32,20 @@ async function prune(type: string) {
           console.error(error);
         }
       });
-
+      break;
+    case 'pods':
+      engines.forEach(async engine => {
+        try {
+          await window.prunePods(engine.id);
```

Comment:
We should probably not wait for the result for better parallelism as there is no feedback